testcontainers/src/core/utils.rs

```diff
@@ -0,0 +1,28 @@
+/// It allows using both runtime flavors without indefinite hanging.
```

Comment:
I'd like to avoid naming something `utils` and in favor put things in files and/or modules that describe the concern of this particular function or macro.

`utils` often leads to being a trash dump.

Reply:
Totally agree